resources/helm/v2.1/istio-control/istio-discovery/templates/service.yaml

```diff
@@ -24,9 +24,6 @@ spec:
     - port: 15014
       name: http-monitoring # prometheus stats
       protocol: TCP
-    - port: 8188
```

Comment:
we stil need this afaik. it's for federation

Reply:
After latest rebase Federation port settings are back.